sql/org_mozilla_firefox/migration/view.sql

```diff
@@ -0,0 +1,23 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.org_mozilla_firefox.migration`
```

Comment:
This will overwrite the current migration ping table for Fennec release. That seems fine for us, but open to alternatives.

Reply:
I would argue this view definition is dangerous. There's no guarantee that these schemas will stay in sync across the different app IDs. In particular, it wasn't possible to make a simple view on top of baseline pings that unions apps because there are a few fields that aren't present everywhere.

Reply:
I think we should verbosely do a union with just the fields we need within the migrated_clients query rather than making this a view that unions them.

Reply:
This ping should remain consistent for two reasons:
1. It's [defined](https://probeinfo.telemetry.mozilla.org/glean/support-migration/pings) in a shared lib
2. Each of these app ids [shares the same repository url](https://probeinfo.telemetry.mozilla.org/glean/repositories)

Reply:
I do think this is a useful view to have for all migrations. I don't see this being a heavily used table aside from me.

Reply:
AFAIK, the reasons you give for consistency of migration ping should also apply to baseline ping. How are the cases different?

Reply:
They do apply for the baseline ping. I made a quick comparison here: https://gist.github.com/fbertsch/9fb62a9d22b45d269555eab3a9f80729, and they _should_ be the same between those apps.

Reply:
Okay, I compared their actual schemas in BQ and they also seem to match using https://gist.github.com/fbertsch/2378a39de9b4cc485d5d9151999f9546.

```
python compare_schemas.py --project moz-fx-data-shared-prod org_mozilla_firefox.baseline org_mozilla_firefox_beta.baseline org_mozilla_fenix.baseline org_mozilla_fennec_aurora.baseline org_mozilla_fenix_nightly.baseline

moz-fx-data-shared-prod.org_mozilla_firefox.baseline and moz-fx-data-shared-prod.org_mozilla_firefox_beta.baseline match schemas
moz-fx-data-shared-prod.org_mozilla_firefox.baseline and moz-fx-data-shared-prod.org_mozilla_fenix.baseline match schemas
moz-fx-data-shared-prod.org_mozilla_firefox.baseline and moz-fx-data-shared-prod.org_mozilla_fennec_aurora.baseline match schemas
moz-fx-data-shared-prod.org_mozilla_firefox.baseline and moz-fx-data-shared-prod.org_mozilla_fenix_nightly.baseline match schemas
```

Reply:
In practice, the baseline schemas are not compatible. If you attempt the following today, you get an error about incompatible types:

```
select * from org_mozilla_fenix_stable.baseline_v1
union all
select * from org_mozilla_firefox_stable.baseline_v1
union all
select * from org_mozilla_firefox_beta_stable.baseline_v1
union all
select * from org_mozilla_fennec_aurora_stable.baseline_v1
union all
select * from org_mozilla_fenix_nightly_stable.baseline_v1
```

It does happen to work for `migration` currently. I won't block this PR on this issue, but be aware that it's technical debt that could cause this ETL to fail unexpectedly in the future.

Reply:
Right, it looks like the structs must not be ordered the same way. This will require close inspection at the pieces of probe-scraper, MSG, and jsonschema-transpiler to figure out where the issue lies for baseline.

Reply:
I opened a [bug 1624862](https://bugzilla.mozilla.org/show_bug.cgi?id=1624862) since this clearly seems like an issue. Thanks for bringing it up!